src/index.html

```diff
@@ -11,7 +11,128 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <h1 class="heading">HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="input-wrapper">
+          Surname:
+          <input
+           type="text"
+           name="surname"
+           autocomplete="off"
+          >
+        </label>
+        <label class="input-wrapper">
+          Name:
+          <input
+           type="text"
+           name="name"
+           autocomplete="off"
+          >
+        </label>
+        <label class="input-wrapper">
+          How old are You?
+          <input
+            type="number"
+             name="age"
+             min="1"
+             max="100"
+             value="12"
```

Comment:
each level of nesting requires **exactly** 2 indents
```suggestion
            type="number"
            name="age"
            min="1"
            max="100"
            value="12"
```